docs/stacks-guide-aws.md

```diff
@@ -0,0 +1,911 @@
+---
+title: "Crossplane Stacks Guide: AWS Setup" toc: true weight: 332 indent: true
+---
+
+# Crossplane Stacks Guide: AWS Setup
+
+## Table of Contents
+
+  - [Crossplane Stacks Guide: AWS Setup](#crossplane-stacks-guide-aws-setup)
+  - [Table of Contents](#table-of-contents)
+  - [Introduction](#introduction)
+  - [Before you begin](#before-you-begin)
+  - [Install AWS stack](#install-aws-stack)
+  - [Configure AWS account](#configure-aws-account)
+  - [Configure Crossplane Provider for
+    AWS](#configure-crossplane-provider-for-aws)
+  - [Set Up Network Configuration](#set-up-network-configuration)
+  - [Configure Provider Resources](#configure-provider-resources)
+  - [Recap](#recap)
+
+## Introduction
+In this guide we are focusing on WordPress workload, and will show how it could
+be deployed using AWS resources. WordPress workload uses a generic Kubernetes
+cluster and a MySQL database as its resource, which are not tied to a specific
+provider (also known as **resource claims**). Instead, these resource claims
+could be configured to use a be satisfied by a specific cloud provider.
+
+We will walk you through installing the AWS stack, configuring an aws account,
+and setting up cloud-specific resources in order to create and configure a
+network configuration in which these cloud resources can communicate with each
+other. Once the network configuration is done, we will specify a resource class
+for each resource claim type. Then we will create WordPress workload, which will
+create related external resources for each claim in AWS.
+
+
+## Before you begin
+Before continuing in this guide, make sure that all the following are true:
+- Crossplane is installed 
+- [crossplane-cli] is installed
+- `kubectl` is configured to target the Crossplane cluster
+- An aws account is available
```

Comment:
Maybe link to how to the setting up aws account doc here

Reply:
Hmm looks like you might actually outline how to do this below, in which case I think we could remove this from the "Before you begin" section

Reply:
good catch!

Reply:
What I mean here, is that the user should have access to an aws account, so it could create a user and configure AWS CLI later. Should I rephrase?